lib/coffee/AccessTokenEncryptor.coffee

```diff
@@ -1,42 +1,61 @@
 crypto = require('crypto')
+async = require('async')
 
 ALGORITHM = 'aes-256-ctr'
 
-keyFn = (password, salt, callback)->
-	return crypto.pbkdf2(password, salt, 10000, 64, 'sha1', callback)
+keyFn = (password, salt, keyLength, callback)->
+	return crypto.pbkdf2(password, salt, 10000, keyLength, 'sha1', callback)
 
 class AccessTokenEncryptor
 
 	constructor: (settings) ->
 
 		@settings = settings
-		@cipherLabel = @settings.cipherLabel
-		throw Error("cipherLabel must not contain a colon (:)") if @cipherLabel?.match(/:/)
+		@cipherLabel = "2019.1"
```

Comment:
We should allow this to come through settings, so each application can have its own current cipherLabel.